epilogue-processor/src/main/java/edu/wpi/first/epilogue/processor/AnnotationProcessor.java

```diff
@@ -40,7 +40,7 @@
   "edu.wpi.first.epilogue.CustomLoggerFor",
   "edu.wpi.first.epilogue.Logged"
 })
-@SupportedSourceVersion(SourceVersion.RELEASE_17)
+@SupportedSourceVersion(SourceVersion.RELEASE_21)
```

Comment:
It might be better to override the method to return SourceVersion.latestSupported(), which would make it automatically use the target version we’re compiling the library for

Reply:
I recall reading somewhere that was dangerous unless the annotation processor was specifically built to make that safe.  But I couldn't find it again.